sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/ShareAsyncClient.java

```diff
@@ -415,16 +462,45 @@ public ShareFileAsyncClient getFileClient(String filePath) {
      */
     public Mono<Response<ShareProperties>> getPropertiesWithResponse() {
         try {
-            return withContext(context -> getPropertiesWithResponse(context));
+            return withContext(context -> getPropertiesWithResponse(new ShareGetPropertiesOptions(), context));
```

Comment:
I think this should call the max public overload rather than the internal one

Reply:
fixed